### PR TITLE
Emojify

### PR DESCRIPTION
**Pull request by n1tr0-5urf3r**
- Emojify wallet commands 
- Added $wallet currencies for a list of all currencies
- Added aliases for old commands

### CODE REVIEW

File: guildwars2/wallet.py [hunk 1]
```diff
@@ -20,6 +21,54 @@ class WalletMixin:
                 pass
             await ctx.send_help(ctx.command)
 
+    def get_emoji_string(self, ctx, name):
+        emoji_string = name.replace(" ", "_").lower()
+        emoji_string = emoji_string.replace("'", "")
+        return self.get_emoji(ctx, emoji_string)
```
Comment (Maselkov) [inline guildwars2/wallet.py]: get_emoji() already does the first part, though what's that part for?
```
emoji_string.replace("'", "")
```

File: guildwars2/wallet.py [hunk 2]
```diff
@@ -20,6 +21,54 @@ class WalletMixin:
...
+
+    async def get_wallet(self, ctx, ids):
+        """Shows key-specific currencies
+
+        Required permissions: wallet
```
Comment (Maselkov) [inline guildwars2/wallet.py]: Help message isn't needed here I think

File: guildwars2/wallet.py (hunk 1 above)
Comment (n1tr0-5urf3r) [reply guildwars2/wallet.py]: this is because of ' in Trader's Key, discord doesnt allow ' in emoji names

File: guildwars2/wallet.py (hunk 2 above)
Comment (n1tr0-5urf3r) [reply guildwars2/wallet.py]: Removed it

File: guildwars2/wallet.py (hunk 1 above)
Comment (Maselkov) [reply guildwars2/wallet.py]: Hmm, just modify get_emoji then if it's an illegal character.

Comment (n1tr0-5urf3r) [reply guildwars2/wallet.py]: Modified get_emoji and removed it from wallet.py

File: guildwars2/wallet.py [hunk 3]
```diff
@@ -131,60 +151,43 @@ class WalletMixin:
...
+        ids_raid = [28, 39]
+        ids_l2 = [79280, 79899, 79469, 80332, 81127, 81706]
+        ids_l3 = [86069, 88955, 86977, 89537, 87645, 90783]
+        cur = await self.get_wallet(ctx, ids_cur)
+        keys = await self.get_wallet(ctx, ids_keys)
+        maps = await self.get_wallet(ctx, ids_maps)
+        token = await self.get_wallet(ctx, ids_token)
+        ls3 = await self.get_ls_currency(ctx, ids_l2)
+        ls4 = await self.get_ls_currency(ctx, ids_l3)
+        raid = await self.get_wallet(ctx, ids_raid)
```
Comment (Maselkov) [inline guildwars2/wallet.py]: That's so many unnecessary API calls. Calling 5 times for wallet and 2 times for account search.

File: guildwars2/wallet.py [hunk 4]
```diff
@@ -131,60 +151,43 @@ class WalletMixin:
         except discord.Forbidden:
             await ctx.send("Need permission to embed links")
 
-    @wallet.command(name="tokens")
+    @wallet.command(name="show", aliases=["keys", "maps", "tokens"])
     @commands.cooldown(1, 5, BucketType.user)
-    async def wallet_tokens(self, ctx):
-        """Shows instance-specific currencies
+    async def wallet_show(self, ctx):
+        """Shows currencies in your wallet
```
Comment (Maselkov) [inline guildwars2/wallet.py]: Change it to "Shows your wallet"

File: guildwars2/wallet.py (hunk 3 above)
Comment (n1tr0-5urf3r) [reply guildwars2/wallet.py]: - Changed get_wallet() and get_ls_currency() to accept a list of lists of ids and call API only once

File: guildwars2/wallet.py (hunk 4 above)
Comment (n1tr0-5urf3r) [reply guildwars2/wallet.py]: - Changed description of $show wallet

**Pull request merged**